MantidQt/CustomInterfaces/test/ReflSaveTabPresenterTest.h

```diff
@@ -33,7 +34,7 @@ class ReflSaveTabPresenterTest : public CxxTest::TestSuite {
   }
   static void destroySuite(ReflSaveTabPresenterTest *suite) { delete suite; }
 
-  ReflSaveTabPresenterTest() {}
+  ReflSaveTabPresenterTest() { FrameworkManager::Instance(); }
```

Comment:
Do we need this? It was removed in the previous commit and then added back when resolving conflicts with master.